src/components/auth/RegisterForm.tsx

```diff
@@ -31,32 +31,32 @@ export default function RegisterForm() {
       <div className={styles.title}>
         INSCRIPT<span className={styles.bluePart}>ION</span>
       </div>
-      <Input value={lastname} onChange={setLastname} onEnter={submit} placeholder="Nom" />
-      <Input value={firstname} onChange={setFirstname} onEnter={submit} placeholder="Prénom" />
-      <Input value={username} onChange={setUsername} onEnter={submit} placeholder="Nom d'utilisateur" />
-      <Input value={password} onChange={setPassword} onEnter={submit} placeholder="Mot de passe" type="password" />
-      <Input
-        value={passwordConfirmation}
-        onChange={setPasswordConfirmation}
-        onEnter={submit}
-        placeholder="Confirmation de mot de passe"
-        type="password"
-      />
+      <a href={`https://cas.utt.fr/cas/login?${new URLSearchParams({
+            service: 'https://etu.assos.utt.fr/login',
+          }).toString()}`} className={styles.cas}>
+        <img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSV1Q_WNZSIZWs328kwIn2Tun59WALMhEesae0jenVHfg&s" alt="Logo UTT" />
```

Comment:
Faut rajouter l'import avec comme dans `LoginForm.tsx`
```suggestion
        <img src={logoutt.src} alt="Logo UTT" />
```